lms/djangoapps/certificates/views/webview.py

```diff
@@ -252,6 +253,17 @@ def _update_course_context(request, context, course, course_key, platform_name):
                                                               '{partner_short_name}.').format(
             partner_short_name=context['organization_short_name'],
             platform_name=platform_name)
+        
+    # Translators: this text describes the course duration
+    course_duration = CourseOverview.get_from_id(course.id).effort
+    if course_duration:
+        if course_duration.strip().isnumeric():
+            num_hour = int(course_duration.strip())
+            context['course_duration'] = translation.ungettext(
```

Comment:
It is correct `ungettext`? Why we don't use `_` from the L55?

Reply:
`ungettext(singular, plural, number)` is used to handle singular and plural forms.
`ugettext(message)` on line L55